grapple/types/pages.py

```diff
@@ -289,6 +299,10 @@ def resolve_pages(self, info, **kwargs):
                 else:
                     pages = pages.filter(content_type=ctype)
 
+            language_code = kwargs.pop("language_code", "")
```

Comment:
Could we use the term "locale" here for consistency with the [API])https://docs.wagtail.org/en/stable/advanced_topics/api/v2/usage.html#filtering-pages-by-locale)?